hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -530,6 +523,11 @@ public void onCompactionCompleted(RocksDB db,
               compactionLogEntry.getOutputFileInfoList(),
               compactionLogEntry.getDbSequenceNumber());
           for (String inputFile : inputFileCompactions.keySet()) {
+            if (!inflightCompactions.containsKey(inputFile)) {
+              LOG.warn("Input file not found in inflightCompactionsMap : {} which should have been added on " +
+                      "compactionBeginListener.",
+                  inputFile);
```

Comment:
- Why doesn't it throw an exception instead?
- What is supposed to do when seeing the warn message?

Reply:
Let me ask it differently -- when seeing this WARN message,
- is it a bug in the code?
- does it mean that some data (snapshot/diff/sst) is corrupted?

> ... However we want to be in a position to debug some other corner case scenario later on ...

Why not printing it in DEBUG? 

The WARN message frequently shows up in OM log and it looks like some kind of data corruption.

Reply:
Could you answer these two simple questions?
- is it a bug in the code?
- does it mean that some data (snapshot/diff/sst) is corrupted?

Reply:
Suggestions
- call remove first and check the return value.
- Use info instead of warn
- Print also the jobInfo.
```java
            final CompactionFileInfo removed = inflightCompactions.remove(inputFile);
            if (removed == null) {
              LOG.info("Input file not found in inflightCompactions: {}, compactionJobInfo is {}", inputFile, ...);
            }
```

Reply:
yeah sure let me make the chang